simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulator.java

```diff
@@ -28,39 +29,45 @@
 import java.lang.System.Logger;
 import java.nio.file.Path;
 
-/** The BlockStreamSimulator class defines the simulator for the block stream. */
+/**
+ * The BlockStreamSimulator class defines the simulator for the block stream.
+ */
```

Comment:
We should not expand these.

Reply:
✅ I reversed the formatting on the docs everywhere.